recipes/packer_pipeline_node.rb

```diff
@@ -34,11 +34,41 @@
   group 'alfred'
 end
 
+# put the key associated with the openstack_taster's users on various clusters
+# so that it can access the instances once created
+node.override['osl-jenkins']['secrets_databag'] = 'osl_jenkins'
+node.override['osl-jenkins']['secrets_item'] = 'jenkins1'
+
+openstack_access_keys = credential_secrets['jenkins']['packer_pipeline']
```

Comment:
And I now I seem to be not able to converge this because:

`Fetching 'base' from git@github.com:osuosl-cookbooks/base (at master)` fails for me.
https://pastebin.osuosl.org/62451/